### DISCUSSION TITLE
Date parsing - forcing timezone

### DISCUSSION THREAD

**Discussion by bryanrcampbell** [js-fns/date-fns#1840] [General]
Hi there - great library, I can really appreciate all the time and thought that went into building date-fns. Two thumbs up for making it immutable!

I've got a 3rd party data source i'm reading in with dates like so: "3/3/2020 0:00:00". I know that these dates are UTC, how do I tell date-fns to parse it as UTC rather than my local timezone?

thanks!

**Comment by bryanrcampbell** [comment]
Any chance anyone has found a way to do this?

**Reply by dwmorrin** [reply]
Did you try [zonedTimeToUtc](https://github.com/marnusw/date-fns-tz#zonedtimetoutc)?

```js
import { parse } from 'date-fns';
import { zonedTimeToUtc } from 'date-fns-tz';

const format = 'M/d/yyyy H:mm:ss';
const customParse = (dateString) =>
  zonedTimeToUtc(parse(dateString, format, new Date()), 'UTC');

customParse('3/3/2020 0:00:00').toISOString(); // "2020-03-03T00:00:00.000Z"
```
